db/structure.sql

```diff
@@ -631,9 +631,9 @@ CREATE TABLE public.external_pull_requests (
     external_project_id bigint NOT NULL,
     created_at timestamp(6) without time zone NOT NULL,
     updated_at timestamp(6) without time zone NOT NULL,
+    number integer,
     opened_at timestamp without time zone,
-    state public.external_pull_request_state,
-    number integer
+    state public.external_pull_request_state
```

Comment:
what are these changes? I don't see any migration added in this PR

Reply:
IDK, seems like it was automatically generated, may be because some local merge I did. Looks likejust changed the order of the lines.